src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,98 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+
+    private Bucket<K, V>[] table;
+    private int size;
+    private int capacity;
+    private int threshold;
+
+    public MyHashMap() {
+        table = new Bucket[DEFAULT_CAPACITY];
+        capacity = DEFAULT_CAPACITY;
+        threshold = (int) (DEFAULT_CAPACITY * LOAD_FACTOR);
+    }
+
+    private class Bucket<K, V> {
+        private final K key;
+        private V value;
+        private Bucket<K, V> next;
+
+        public Bucket(K key, V value) {
+            this.key = key;
+            this.value = value;
+        }
+    }
```

Comment:
usually, inner class is created at the end of the file

Reply:
I am not sure whether it is obligatory, still. accepted)